reference/5.1/Microsoft.PowerShell.Core/About/about_Module_Manifest.md

```diff
@@ -0,0 +1,1714 @@
+# DotNetFrameworkVersion = ''
+# CLRVersion = ''
+# ProcessorArchitecture = ''
+# RequiredModules = @()
+# RequiredAssemblies = @()
+# ScriptsToProcess = @()
+# TypesToProcess = @()
+# FormatsToProcess = @()
+# NestedModules = @()
+FunctionsToExport = @()
+CmdletsToExport = @()
+VariablesToExport = '*'
+AliasesToExport = @()
+# DscResourcesToExport = @()
+# ModuleList = @()
+# FileList = @()
+PrivateData = @{
+    PSData = @{
+        # Tags = @()
+        # LicenseUri = ''
+        # ProjectUri = ''
+        # IconUri = ''
+        # ReleaseNotes = ''
+    } # End of PSData hashtable
+} # End of PrivateData hashtable
+# HelpInfoURI = ''
+# DefaultCommandPrefix = ''
+}
+```
+
+## Manifest settings
+
+The following sections detail every available setting in a module manifest
+and how you can use them. They start with the setting's type definition which
+lists:
+
+- **Type**: The object type that is expected. The value in the manifest does not
+  need to be specified as the type but it must be convertible to that type. For
+  example, `'1.2.3'` can be converted to **System.Version** but `'a.b.c'`can't.
+- **Aliases**: The alternate names this setting can have, if any.
+- **Required**: Whether this setting is required for `Import-Module`.
+- **Default value**: What value, if any, will be used when this setting isn't
+  set. A value of `None` indicates that the setting will be ignored if not set.
+- **Accept wildcard characters** - Whether this setting can take a wildcard
+  value or not.
+
+### RootModule
+
+```yaml
+Type: System.String
+Aliases: ModuleToProcess
+Required: False
+Default value: None
+Accept wildcard characters: False
+```
+
+This setting the primary or root file of the module. When the module is
+imported, the members that are exported from the root module file are imported
+into the caller's session state.
+
+The value must be the file path to one of the following:
+
+- a script (`.ps1`)
+- a script module (`.psm1`)
+- a module manifest (`.psd1`)
+- an assembly (`.dll`)
+- a cmdlet definition XML file (`.cdxml`)
+- a PowerShell Workflow (`.xaml`)
+
+The file path should be relative to the module manifest.
+
+If a module has a manifest file and no root file was designated in the
+**RootModule** key, the manifest becomes the primary file for the module, and
+the module becomes a manifest module (ModuleType = Manifest). When
+**RootModule** is defined, the module's type is determined from the file
+extension used:
+
+- a `.ps1` or `.psm1` file makes the module type **Script**
+- a `.psd1` file makes the module type **Manifest**
+- a `.dll` file makes the module type **Binary**
+- a `.cdxml` file makes the module type **CIM**
+- a `.xaml` file makes the module type **Workflow**
+
+To export members from `.psm1` or `.dll` files in a module that has a manifest,
+the names of those files must be specified in the values of the **RootModule**
+or **NestedModules** keys in the manifest. Otherwise, their members aren't
+exported.
```

Comment:
What about cdxml and xaml types, are their values exported differently? In script-based modules, you can use the `Export-ModuleMember` cmdlet.

So really, you mean that for exports defined in the manifest, the files containing the items to be exported must be listed in **RootModule** or **NestModules**.